prettified single file tsx version

rvstowrn opened the pull request:
# Description

Created a non breaking prettified tsx version.

- New feature (non-breaking change which adds functionality)
- I have performed a self-review of my own code
- My changes generate no new warnings
- Any dependent changes have been merged and published in downstream modules

sriram-kailasam commented on public/htmlTest/html.html:
```diff
@@ -0,0 +1,201 @@
+
+
+
+		// var k=10;
```
No commented code

sriram-kailasam commented on public/htmlTest/html.html:
```diff
@@ -0,0 +1,201 @@
+		var canvas=document.getElementById('canvas');
+		canvas.width = 600;
+		canvas.height = 600;
+		// var c=canvas.getContext('2d');
```
No commented code

sriram-kailasam commented on src/styles.css:
```diff
@@ -0,0 +1,23 @@
+}
+canvas{
+	box-shadow: 0 0 5px gray;
+	/*width: 300px;*/
```
No commented code

sriram-kailasam commented on public/htmlTest/html.html:
```diff
@@ -0,0 +1,201 @@
+    canvas.addEventListener('touchend', handleMouseUp, false);
+
+    function distanceBetween(point1, point2) {
+    return Math.sqrt(Math.pow(point2.x - point1.x, 2) + Math.pow(point2.y - point1.y, 2));
```
I dont think Prettier has worked here

sriram-kailasam commented on public/htmlTest/html.html:
```diff
@@ -0,0 +1,201 @@
+
+
+
+$('#btn').click(function() {
```
Button id name can be improved

sriram-kailasam commented on src/index.tsx:
```diff
@@ -0,0 +1,236 @@
+    canvas.addEventListener("mouseup", handleMouseUp.bind(this), false);
+    canvas.addEventListener("touchend", handleMouseUp.bind(this), false);
+
+    function distanceBetween(point1: any, point2: any) {
```
No point in using Typescript if you'll make everything any

sriram-kailasam commented on src/index.tsx:
```diff
@@ -0,0 +1,236 @@
+    }
+
+    var pstate: any;
+    var plvalx: any;
```
What's the type?

sriram-kailasam commented on src/index.tsx:
```diff
@@ -0,0 +1,236 @@
+      e.preventDefault();
+
+      if (this.state.choice === "rectangle") {
+        if (prevjob.done === false) {
```
!prevjob.done

Dont use === for booleans

sriram-kailasam commented on src/index.tsx:
```diff
@@ -0,0 +1,236 @@
+        x: any,
+        y: any;
+
+      for (var i: any = 0; i < dist; i++) {
```
No need to specify any type here. Just leave it as it is, it's clear that it is a number

sriram-kailasam commented on src/index.tsx:
```diff
@@ -0,0 +1,236 @@
+
+          <button
+            onClick={() => {
+              var link: any = document.createElement("a");
```
Use const wherever possible

sriram-kailasam commented on src/index.tsx:
```diff
@@ -0,0 +1,236 @@
+              link.click();
+            }}
+          >
+            Downlaod
```
Spelling

sriram-kailasam commented on src/index.tsx:
```diff
@@ -0,0 +1,236 @@
+import { SketchPicker } from "react-color";
+import "./styles.css";
+
+class CanvasComponent extends React.Component {
```
Why is this component inside index.tsx? This should go in another file

sriram-kailasam commented on src/index.tsx:
```diff
@@ -0,0 +1,236 @@
+    ctx.fillStyle = "#ffffff";
+    ctx.fill();
+    canvas.addEventListener("mouseout", handleMouseOut.bind(this), false);
+    canvas.addEventListener("mousedown", handleMouseDown.bind(this), false);
```
These could have been arrow functions, then you won't have to use bind

sriram-kailasam commented on src/index.tsx:
```diff
@@ -0,0 +1,236 @@
+    }
+
+    function handleMouseDown(e: any) {
+      isDrawing = true;
```
This should have been inside state. Any reason why it is isn't?

rvstowrn commented:
Corrected spelling error.
Made a separate file for canvas component.
Used arrow functions.
Used const variables wherever possible and given types to significant variables.

sriram-kailasam commented:
@rvstowrn Please follow this commit message format: https://gist.github.com/joshbuchea/6f47e86d2510bce28f8e7f42ae84c716

Also, make your commit messages more descriptive. "made required changes" is not an acceptable commit message. If I see the history of commits, I won't be able to tell what changes were requested and which were made.

rvstowrn commented:
> 
> 
> @rvstowrn Please follow this commit message format: https://gist.github.com/joshbuchea/6f47e86d2510bce28f8e7f42ae84c716
> 
> Also, make your commit messages more descriptive. "made required changes" is not an acceptable commit message. If I see the history of commits, I won't be able to tell what changes were requested and which were made.

Will make correct commit messages from now on. For this one I have made an empty commit with a proper message describing the previous commit so that one could tell what changes where made in previous commit while viewing the history of commits.

sriram-kailasam commented on src/Components/CanvasComponent.tsx:
```diff
@@ -0,0 +1,243 @@
+    ) as HTMLCanvasElement;
+    var canvasWidth: number = canvas.width;
+    var canvasHeight: number = canvas.height;
+    var ctx: any = canvas.getContext("2d");
```
No need to specify type if you are going to make it `any`

sriram-kailasam commented on src/Components/CanvasComponent.tsx:
```diff
@@ -0,0 +1,243 @@
+    var canvasWidth: number = canvas.width;
+    var canvasHeight: number = canvas.height;
+    var ctx: any = canvas.getContext("2d");
+    var jobStart: boolean = false;
```
First try to use `const`. If you feel the need to change the variable, use `let`. `var` should not be used at all in most cases, because of its weird scope.

sriram-kailasam commented on src/Components/CanvasComponent.tsx:
```diff
@@ -0,0 +1,243 @@
+      return { x: mx, y: my };
+    }
+
+    let handleMouseOut = (e: any) => {
```
Either remove the type, or use HTMLEvent.

sriram-kailasam commented on src/Components/CanvasComponent.tsx:
```diff
@@ -0,0 +1,243 @@
+
+          <button
+            onClick={() => {
+              const link: any = document.createElement("a");
```
No need to specify type.

rvstowrn commented:
Removed any data type from all previous any type variables except in ctx variable
Used const and let in all variables 
Removed number as data type where variables were initialized number value already
